pyntc/devices/jnpr_device.py

```diff
@@ -116,22 +117,36 @@ def close(self):
         if self.connected:
             self.native.close()
 
-    def config(self, command, format="set"):
-        try:
-            self.cu.load(command, format=format)
-            self.cu.commit()
-        except ConfigLoadError as e:
-            raise CommandError(command, e.message)
+    def config(self, commands, format="set"):
+        """Send configuration commands to a device.
 
-    def config_list(self, commands, format="set"):
+        Args:
+             commands (str, list): String with single command, or list with multiple commands.
+
+         Raises:
+             ConfigLoadError: Issue with loading the command.
+             CommandListError: Issue with a command in the list provided.
+        """
+        if isinstance(commands, str):
+            commands = [commands]
         try:
             for command in commands:
                 self.cu.load(command, format=format)
-
             self.cu.commit()
         except ConfigLoadError as e:
             raise CommandListError(commands, command, e.message)
```

Comment:
This should check to see if `commands` was passed as a list or a string, and then raise either `CommandError` or `CommandListError` as appropriate.